tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/types/json/BJSONValueTest.java

```diff
@@ -410,7 +410,7 @@ public void testJsonArrayToJsonCasting() {
         Assert.assertEquals(returns[0].stringValue(), "[[1, 2, 3], [3, 4, 5], [7, 8, 9]]");
     }
 
-    @Test
+    @Test(groups = {"broken"})
```

Comment:
What happened with this test?

Reply:
org.ballerinalang.util.exceptions.BLangRuntimeException: error: assertion error: expected 'json[][]', found 'json[]' {}
	at testJsonToJsonArrayCasting(json-value.bal:321)